### ISSUE TITLE
Player can't sell their items.

### ISSUE THREAD

**Issue by SN-Koarashi** [EpiCanard/GlobalMarketChest#92]
Issue goal: Bug
Platform: Spigot 1.14.4
Plugin version: GlobalMarketChest v1.3.1

**Description**
Player can't sell their items.
Throw error in console following:
```
[20:37:08] [Server thread/WARN]: java.lang.NoSuchMethodException: net.minecraft.server.v1_14_R1.MinecraftKey.b()
[20:37:08] [Server thread/WARN]: 	at java.lang.Class.getMethod(Unknown Source)
[20:37:08] [Server thread/WARN]: 	at fr.epicanard.globalmarketchest.utils.reflection.VersionSupportUtils.invokeMethod(VersionSupportUtils.java:91)
[20:37:08] [Server thread/WARN]: 	at fr.epicanard.globalmarketchest.utils.reflection.VersionSupportUtils.getMinecraftKey(VersionSupportUtils.java:186)
[20:37:08] [Server thread/WARN]: 	at fr.epicanard.globalmarketchest.utils.ItemStackUtils.getMinecraftKey(ItemStackUtils.java:76)
[20:37:08] [Server thread/WARN]: 	at fr.epicanard.globalmarketchest.utils.ItemStackUtils.isBlacklisted(ItemStackUtils.java:257)
[20:37:08] [Server thread/WARN]: 	at fr.epicanard.globalmarketchest.gui.shops.interfaces.CreateAuctionItem.onDrop(CreateAuctionItem.java:191)
[20:37:08] [Server thread/WARN]: 	at fr.epicanard.globalmarketchest.listeners.GUIListener.onClick(GUIListener.java:38)
[20:37:08] [Server thread/WARN]: 	at sun.reflect.GeneratedMethodAccessor106.invoke(Unknown Source)
[20:37:08] [Server thread/WARN]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[20:37:08] [Server thread/WARN]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[20:37:08] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:316)
[20:37:08] [Server thread/WARN]: 	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[20:37:08] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:529)
[20:37:08] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:514)
[20:37:08] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:2133)
[20:37:08] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.PacketPlayInWindowClick.a(SourceFile:33)
[20:37:08] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.PacketPlayInWindowClick.a(SourceFile:10)
[20:37:08] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:19)
[20:37:08] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.TickTask.run(SourceFile:18)
[20:37:08] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeTask(SourceFile:144)
[20:37:08] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23)
[20:37:08] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeNext(SourceFile:118)
[20:37:08] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.MinecraftServer.aX(MinecraftServer.java:910)
[20:37:08] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.MinecraftServer.executeNext(MinecraftServer.java:903)
[20:37:08] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeAll(SourceFile:103)
[20:37:08] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.MinecraftServer.sleepForTick(MinecraftServer.java:886)
[20:37:08] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:820)
[20:37:08] [Server thread/WARN]: 	at java.lang.Thread.run(Unknown Source)
[20:37:08] [Server thread/ERROR]: Could not pass event InventoryClickEvent to GlobalMarketChest v1.3.1
org.bukkit.event.EventException: null
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:320) ~[spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:529) ~[spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:514) ~[spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:2133) ~[spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at net.minecraft.server.v1_14_R1.PacketPlayInWindowClick.a(SourceFile:33) ~[spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at net.minecraft.server.v1_14_R1.PacketPlayInWindowClick.a(SourceFile:10) ~[spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at net.minecraft.server.v1_14_R1.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:19) ~[spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at net.minecraft.server.v1_14_R1.TickTask.run(SourceFile:18) [spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeTask(SourceFile:144) [spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) [spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeNext(SourceFile:118) [spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at net.minecraft.server.v1_14_R1.MinecraftServer.aX(MinecraftServer.java:910) [spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at net.minecraft.server.v1_14_R1.MinecraftServer.executeNext(MinecraftServer.java:903) [spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeAll(SourceFile:103) [spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at net.minecraft.server.v1_14_R1.MinecraftServer.sleepForTick(MinecraftServer.java:886) [spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:820) [spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_161]
Caused by: java.lang.NullPointerException
	at fr.epicanard.globalmarketchest.utils.reflection.VersionSupportUtils.getMinecraftKey(VersionSupportUtils.java:186) ~[?:?]
	at fr.epicanard.globalmarketchest.utils.ItemStackUtils.getMinecraftKey(ItemStackUtils.java:76) ~[?:?]
	at fr.epicanard.globalmarketchest.utils.ItemStackUtils.isBlacklisted(ItemStackUtils.java:257) ~[?:?]
	at fr.epicanard.globalmarketchest.gui.shops.interfaces.CreateAuctionItem.onDrop(CreateAuctionItem.java:191) ~[?:?]
	at fr.epicanard.globalmarketchest.listeners.GUIListener.onClick(GUIListener.java:38) ~[?:?]
	at sun.reflect.GeneratedMethodAccessor106.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_161]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_161]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:316) ~[spigot-1.14.4.jar:git-Spigot-9de398a-9c887d4]
	... 17 more
```

**Comment by EpiCanard** [comment]
Thanks to report the bug. The last tested version was 1.14.3. It seems there were breaking changes with this last version.

I will try to fix this bug

**Comment by SN-Koarashi** [comment]
Thanks.

**Comment by EpiCanard** [comment]
This error appends with all items ?

It's seems to be caused by the items blacklisted. Some of items inside config variable `ItemsBlacklist` may have changed of name.

I don't have a computer right now to test.

**Comment by SN-Koarashi** [comment]
I remove of all in ItemsBlacklist.
It still throw error in console

**Comment by EpiCanard** [comment]
Ok, so it's something else.

**Comment by EpiCanard** [comment]
Fixed in last version 1.3.2

**Comment by EpiCanard** [comment]
The bug is fixed but you still can have a stack trace error, just a little mistake will be fix in next release.
